lib/rubocop/runner.rb

```diff
@@ -27,7 +27,11 @@ def initialize(options, config_store)
 
     def run(paths)
       target_files = find_target_files(paths)
-      inspect_files(target_files)
+      if @options[:list_target_files]
+        puts target_files.map { |file| PathUtil.relative_path(file) }
```

Comment:
I kinda do too, but I feel guilty about increasing the max length by too much haha.

(I like questions of taste. They're what brought us together)